src/app/components/dialogs/invalidParametersDialog/invalidParametersDialog.component.html

```diff
@@ -0,0 +1,20 @@
+<app-base-dialog [data]="data">
+
+  <div class="container">
+    <h1>The data could not be loaded</h1>
```

Comment:
- h1 might be too big here.  It seems massive!
- The text can be passed into the app-base-dialog anyway, I think at title="The data could not be loaded"